src/js/modules/plant/model.js

```diff
@@ -12,17 +12,37 @@ export default Model.extend({
     userActivity: false,
   },
 
-  getProjection: function() {
+  getProjection() {
     return this.manifesto()
       .getProjectionsFor(this.get('objectId'))[
         this.get('currentProjection') || this.get('projection') || 0
       ];
   },
 
-  setProjection: function(at_) {
+  setProjection(at_) {
     const projections = this.get('projections');
     let at = at_ > projections.length ? 0 : at_;
     at = at < 0 ? projections.length : at;
     this.set('projection', at);
   },
+
+  setPosX({ x, width }) {
```

Comment:
I don't know if the better aproach here will not be to create just two functions: `setPosition` and `getPosition` that will accept/return object with `x` and `y` and instead passing `width` and `height` maybe we should pass whole `overlay` object here? What do You think?